.changeset/orange-feet-share.md

```diff
@@ -0,0 +1,8 @@
+---
+"chainlink": minor
+---
+
+Implemented new EVM Multinode design. The Multinode is now called by chain clients to retrieve the best healthy RPC rather than performing RPC calls directly.
```

Comment:
> The Multinode is now called by chain clients to retrieve the best healthy RPC rather than performing RPC calls directly.

Can you elaborate on this? The old code already did a variation of this AFAIK, so what exactly changed?

Reply:
For example, it would be incorrect to route every individual RPC call to the current healthiest node, because it is important to maintain consistency across queries (same reason why load balancing can RPCs can be harmful). What are the new conditions that cause a change in RPC?

Reply:
Updated the changeset to better outline the purpose of this PR. The RPC selection conditions were not a focus of this PR. Instead the focus is on enabling non-EVM chains to use MultiNode.